CONTRIBUTING.md

```diff
@@ -155,6 +162,62 @@ git push origin HEAD # This will close the PR
 * Keeps commit history clean and easy to read
 
 
+## App Contribution Guidelines
+
+This section details the app architecture, conventions, and best practices.
+Please read this section before making contributions within the `app` directory.
+
+### Overview
+
+Please start by reading [`app/README.md`](app//README.md) for a high-level overview of the app and how to get started developing with it.
+
+### Composition API
+
+The app is written with the Vue 3 Composition API because it's much better at keeping code organized (especially for larger codebases), and because it provides better TypeScript support.
+All pages and components should be written using the Composition API, and not the standard Options API.
+If you are unfamiliar with the Composition API, you can read more about it [here](https://v3.vuejs.org/guide/composition-api-introduction.html#why-composition-api).
+
+Check out [`GrantRegistryNewGrant.vue`](app/src/views/GrantRegistryNewGrant.vue) for an example of a component that uses the Composition API.
+In the Composition API docs referenced earlier, most examples put everything within the `setup()` method.
+This is ok, but typically should only be done for simple components. For more complex components, logic and variables should be moved in composition functions, like the `useNewGrant()` method defined in `GrantRegistryNewGrant.vue`.
```

Comment:
Suggested simplification for the first sentence: "This is ok for simple components."

Reply:
Done in 42a8e96d95f2138fea0243f7438bb8c5d7f4f6a6